tvjs/src/DirectionalNavigation/directionalnavigation-1.0.0.0.js

```diff
@@ -819,7 +819,12 @@
         }
     });
 
-    document.addEventListener("DOMContentLoaded", function () {
+    var _initRun = false;
+    var init = function () {
```

Comment:
To follow the convention, could you name this function with an underscore? "_init".